rollup.config.js

```diff
@@ -0,0 +1,49 @@
+import postcss from 'rollup-plugin-postcss';
```

Comment:
This is a WIP.

Still need to generate all of umd, esm, cjs (dev and minified versions).

`rollup -cw` works and serve the ex on localhost:3000